docs/source/changelog.rst

```diff
@@ -8,6 +8,7 @@ Changelog
     * Fixes
         * Fix performance regression in DFS (:pr:`637`)
         * Fix deserialization of feature relationship path (:pr:`665`)
+        * Fix variable types modify in Entity init (:pr:`675`)
```

Comment:
Can this message convey that we were fixing the user-supplied variable being modified?

Reply:
Fixed.